deps.edn

```diff
@@ -1,26 +1,63 @@
 {:paths ["src"]
- :deps {com.draines/postal                      {:mvn/version "2.0.5"},
-        com.flexiana/tiny-rbac                  {:mvn/version "0.1.1"},
-        com.taoensso/timbre                     {:mvn/version "5.1.2"},
-        crypto-password/crypto-password         {:mvn/version "0.3.0"},
-        funcool/cuerdas                         {:mvn/version "2.2.1"},
-        info.sunng/ring-jetty9-adapter          {:mvn/version "0.17.5"},
-        metosin/malli                           {:mvn/version "0.8.4"},
-        metosin/reitit                          {:mvn/version "0.5.18"},
-        migratus/migratus                       {:mvn/version "1.3.6"},
-        nilenso/honeysql-postgres               {:mvn/version "0.4.112"},
-        org.clojure/core.async                  {:mvn/version "1.5.648"}
-        org.clojure/data.json                   {:mvn/version "2.4.0"},
-        org.clojure/data.xml                    {:mvn/version "0.0.8"},
-        org.postgresql/postgresql               {:mvn/version "42.3.3"},
-        piotr-yuxuan/closeable-map              {:mvn/version "0.35.0"},
-        seancorfield/next.jdbc                  {:mvn/version "1.2.659"},
-        yogthos/config                          {:mvn/version "1.2.0"}}
+ :deps  {org.clojure/clojure             {:mvn/version "1.11.1"}
+         org.clojure/core.async          {:mvn/version "1.5.648"}
+         org.clojure/tools.cli           {:mvn/version "1.0.206"}
+         org.clojure/data.json           {:mvn/version "2.4.0"},
+         org.clojure/data.xml            {:mvn/version "0.0.8"},
+
+         com.draines/postal              {:mvn/version "2.0.5"},
+         com.flexiana/tiny-rbac          {:mvn/version "0.1.1"},
+         com.taoensso/timbre             {:mvn/version "5.2.1"},
+         crypto-password/crypto-password {:mvn/version "0.3.0"},
+         funcool/cuerdas                 {:mvn/version "2.2.1"},
+         info.sunng/ring-jetty9-adapter  {:mvn/version "0.17.5"},
+         metosin/malli                   {:mvn/version "0.8.4"},
+         metosin/reitit                  {:mvn/version "0.5.18"},
+         ;; migratus/migratus               {:mvn/version "1.3.7"},
+         migratus/migratus               {:git/url "https://github.com/Flexiana/migratus"
+                                          :sha     "fba4aa9d57da6cc6a1d6e8c4a2b4198ad5a44315"}
+         nilenso/honeysql-postgres       {:mvn/version "0.4.112"},
+         org.postgresql/postgresql       {:mvn/version "42.3.3"},
+         piotr-yuxuan/closeable-map      {:mvn/version "0.35.0"},
+         seancorfield/next.jdbc          {:mvn/version "1.2.659"},
+         yogthos/config                  {:mvn/version "1.2.0"}}
+
  :aliases
  {:dev
   {:extra-paths ["config/dev"]}
 
+  :repl
+  {:extra-deps
+   {cider/cider-nrepl {:mvn/version "0.28.4"}
```

Comment:
Shouldn't these be somewhere personal to the developer? Is there an equivalent to `~/.lein/.profiles.clj` we can use?

Reply:
Of course it is personal because repl methodology is personal :) and yes there is an equivalent to  to profiles.clj (.config/clojure/deps.edn on XDG linux).
I will delete this alias